README.md

```diff
@@ -5,6 +5,11 @@ The texts were prepared to seemlessly integrate with the DraCor pipeline. Networ
 
 We also added Wikidata IDs per play.
 
+# About this fork:
+The tokenization and linguistic mark-up was removed with `remove_w_pc_markup.xsl` to reduce filesize. 
+
+**Attention**: This corpus is intended to be used within the DraCor infrastructure for research questions that need network- and/or structural data (acts, sceens, ..) only.
+
```

Comment:
Since this is going to end up in the official _dracor-org/shakedracor_  these paragraphs could probably be rephrased not to mention a fork.

Also, since it is quite a significant change to the original sources, it could make sense to include this information just after "The texts were prepared to seemlessly integrate with the DraCor pipeline." above.